test/unitTest/java/org/thoughtcrime/securesms/crypto/FingerprintTestAuthentication.java

```diff
@@ -0,0 +1,40 @@
+package org.thoughtcrime.securesms.crypto;
+
+import android.content.Context;
+
+import org.junit.Test;
+import org.powermock.api.mockito.PowerMockito;
+import org.thoughtcrime.securesms.BaseUnitTest;
+import org.thoughtcrime.securesms.FingerprintAuthenticationHandler;
+import org.thoughtcrime.securesms.util.FingerprintAuthenticationUtil;
+
+import java.security.KeyPairGenerator;
+import java.security.KeyStore;
+
+import javax.crypto.Cipher;
+
+import static junit.framework.Assert.assertFalse;
+
+
+public class FingerprintTestAuthentication extends BaseUnitTest {
+    @Test
+    public void cipherTest() {
+        Context mockContext = PowerMockito.mock(Context.class);
```

Comment:
Don't need to mock a context if you're inheriting from the base unit test; there's a context in there already mocked